internal/plan/plan.go

```diff
@@ -89,6 +95,44 @@ func (s *Service) Equal(other *Service) bool {
 	return reflect.DeepEqual(s, other)
 }
 
+// ParseBackoff returns the parsed version of the backoff duration list, or
+// the default if it's not specified.
+func (s *Service) ParseBackoff() ([]time.Duration, error) {
+	if s.Backoff == nil {
+		return []time.Duration{
+			0,
+			10 * time.Second,
+			20 * time.Second,
+			40 * time.Second,
+			80 * time.Second,
+			160 * time.Second,
+			320 * time.Second,
+		}, nil
+	}
+	return parseDurationList(*s.Backoff)
+}
+
+func parseDurationList(strings []string) ([]time.Duration, error) {
+	durations := make([]time.Duration, len(strings))
+	for i, s := range strings {
+		d, err := time.ParseDuration(s)
+		if err != nil {
+			return nil, err
+		}
+		durations[i] = d
+	}
+	return durations, nil
+}
+
+// ParseStartTime returns the parsed version of the start time duration, or
+// the default if it's not specified.
+func (s *Service) ParseStartTime() (time.Duration, error) {
```

Comment:
Seems better to have this as a ParseTime function of the plan package, instead of expecting every time attribute to come with an analogous function.

That said, why are we holding those values as strings instead of parsing them together with the rest of the service, and having the right values (including the default) in place at all times?

Reply:
As mentioned above, this is gone now as it's parsed directly into a duration in `plan.Service`.